src/clr-angular/utils/smart-popover/providers/smart-popover-events.service.ts

```diff
@@ -109,8 +119,8 @@ export class ClrSmartPopoverEventsService implements OnDestroy {
   private escapeListener: () => void;
   public addEscapeListener() {
     this.escapeListener = this.renderer.listen(this.document, 'keydown.escape', event => {
-      this.setAnchorFocus();
       this.smartOpenService.open = false;
+      this.setAnchorFocus();
```

Comment:
Something small that we forgot - please add this to the subscription inside `addScrollListener` too.
Now on scroll focus goes to the body, as it used to do with the Esc key.

Reply:
Thanks for catching this. I'll add it.